src/app/core/common-components/entity-form/entity-form.service.ts

```diff
@@ -111,42 +120,63 @@ export class EntityFormService {
    * @param forTable
    */
   public createFormGroup<T extends Entity>(
-    formFields: FormFieldConfig[],
+    formFields: ColumnConfig[],
     entity: T,
     forTable = false,
   ): EntityForm<T> {
-    this.extendFormFieldConfig(formFields, entity.getConstructor(), forTable);
     const formConfig = {};
-    const entitySchema = entity.getSchema();
     const copy = entity.copy();
-    formFields
-      .filter((formField) => entitySchema.get(formField.id))
```

Comment:
Do you know why this filter is not needed anymore?

Reply:
I think I had assumed because we don't rely on the schema here anymore, we do not need to explicitly remove them. But actually there may have been more reason behind filtering them, like the form not being relevant for those that are not related to entity properties ...
I am putting this back in, thanks :+1: